src/Ampersand/Input/ADL1/CtxError.hs

```diff
@@ -174,9 +160,9 @@ mkMultipleRootsError roots gs
              ] ++
              [ "  - "++showA x++" at "++showFullOrig (origin x) | x<-NE.toList gs]++
              [ "Parhaps you could add the following statements:"]++
-             [ "  CLASSIFY "++name cpt++" ISA "++show rootName    | cpt<-roots]
+             [ "  CLASSIFY "++showWithAliases cpt++" ISA "++show rootName    | cpt<-roots]
```

Comment:
Are you sure that `show rootName` is not one of `showWithAliases cpt`?

Reply:
Yes, it is part of the definition of the function `rootName`:
~~~
rootName = L.intercalate "_Or_" . map showWithAliases $ roots
~~~